frontend/desktop/src/components/common/TemplateCanvas/NodeTemplate/ShortcutPanel.vue

```diff
@@ -0,0 +1,155 @@
+/**
+* Tencent is pleased to support the open source community by making 蓝鲸智云PaaS平台社区版 (BlueKing PaaS Community
+* Edition) available.
+* Copyright (C) 2017-2019 THL A29 Limited, a Tencent company. All rights reserved.
+* Licensed under the MIT License (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+* http://opensource.org/licenses/MIT
+* Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+* an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations under the License.
+*/
+<template>
+    <div
+        ref="shortcutWrap"
+        v-if="idOfNodeShortcutPanel === node.id"
+        class="shortcut-panel">
+        <ul class="shortcut-wrap">
+            <li v-if="isShowConfigIcon" @click.stop="onShowNodeConfig" class="shortcut-item common-icon-gear"></li>
+            <li @click="onAppendNode('tasknode')" class="shortcut-item common-icon-node-tasknode"></li>
+            <li @click="onAppendNode('subflow')" class="shortcut-item common-icon-node-subflow"></li>
+            <li @click="onAppendNode('parallelgateway')" class="shortcut-item common-icon-node-parallelgateway"></li>
+            <li @click="onAppendNode('branchgateway')" class="shortcut-item common-icon-node-branchgateway"></li>
+            <li @click="onAppendNode('convergegateway')" class="shortcut-item common-icon-node-convergegateway"></li>
```

Comment:
`v-for` ['tasknode', 'subflow', 'parallelgateway', 'branchgateway', 'convergegateway']